### ISSUE TITLE
Primordial pearl can have mending

### ISSUE THREAD

**Issue by tyler489** [Azanor/thaumcraft-beta#1102]
Pretty sure that is not intended feature....

**Comment by aycfes** [comment]
agreed, but i personally think it should be, with one drawback; if you repair beyond "full" (could probably be done by adding one more durability to it) it goes kablamo and blows up quite a large radius
